recipes/cerbere/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "2.1.0" %}
+
+package: 
+  name: cerbere
+  version: {{ version }}
+
+source:
+  url: https://gitlab.ifremer.fr/cerbere/cerbere/-/archive/V2.1.0/cerbere-{{ version }}.tar.gz
+  sha256: de3aae083cca91381b642a10282c9718849b2a46beda596c67cb8eb8ceb7ebee
+
+build:
+  number: 0
+  skip: true
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+
+  run:
+    - python
```

For noarch, need to put Python version range here:
```suggestion
    - python >=3.6
    - pip
    - wheel

  run:
    - python >=3.6
```